web/thesauruses/clojure/functions.json

```diff
@@ -0,0 +1,68 @@
+{
+  "meta": {
+    "language": "Clojure",
+    "language_version": "1.10.3",
+    "language_name": "Clojure"
+  },
+  "categories": {
+    "Void Functions": [
+      "void_function_no_parameters",
+      "void_function_with_parameters",
+      "void_function_variable_parameters"
+    ],
+    "Return Value Functions": [
+      "return_value_function_no_parameters",
+      "return_value_function_with_parameters",
+      "return_value_function_variable_parameters"
+    ],
+    "Lambda/Anonymous Functions": [
+      "anonymous_function_no_parameters",
+      "anonymous_function_with_parameters",
+      "anonymous_function_variable_parameters"
+    ],
+    "Subroutines": [
+      "call_subroutine",
+      "return_from_subroutine"
+    ]
+  },
+  "functions": {
+    "void_function_no_parameters": {
+      "code": "(defn void-no-params []\n  (println \"Function called\"))\n\n;; call it\n(void-no-params)\nFunction called",
```

Comment:
This is just to define the function. You can remove the part about calling it (that's below in this file). And same for the other concepts below.

Reply:
OK, got rid of all the "call it" stuff except for under the subroutines header.

Reply:
You still have "call it\n(void-no-params)\nFunction called" in there though, which isn't compilable code. Just show how to define the function, don't show how to call it.